Python/compile.c

```diff
@@ -7332,6 +7329,9 @@ label_exception_targets(basicblock *entryblock) {
                 }
             }
             else {
+                if (instr->i_opcode == YIELD_VALUE) {
+                    instr->i_oparg = except_stack->depth;
+                }
```

Comment:
I'm just trusting that this is the right place to insert this. @iritkatriel ?

Reply:
I can't think of a better place. We don't have the except stack depth anywhere else.